corehq/apps/users/role_utils.py

```diff
@@ -1,28 +1,13 @@
-from corehq.apps.users.models import Permissions, UserRole, UserRolePresets
-
-
-def get_or_create_role_with_permissions(domain, name, permissions):
```

Comment:
this function was only used by `initialize_domain_with_default_roles` which is only called when creating a new domain. It seems unnecessary to check for existing roles with matching permissions when setting up the roles for a new domain.